hypervisor/arch/x86/guest/vmexit.c

```diff
@@ -409,7 +409,7 @@ static int32_t wbinvd_vmexit_handler(struct acrn_vcpu *vcpu)
 
 	/* GUEST_FLAG_RT has not set in post-launched RTVM before it has been created */
 	if ((!is_software_sram_enabled()) && (!has_rt_vm())) {
-		cache_flush_invalidate_all();
+		wbinvd();
 	} else {
 		if (is_rt_vm(vcpu->vm)) {
 			walk_ept_table(vcpu->vm, ept_flush_leaf_page);
```

Comment:
This function (to flush all) for both SSRAM case or not, can be a seperate wrap API.

Reply:
I think this function would have large logic change when we support multi-RTVM. In that time, we could hide SSRAM from vmexit module.

Reply:
OK. But call flush_cache_all is better IMO